packages/transformer/src/DetachedExportElementAspectsStrategy.ts

```diff
@@ -0,0 +1,118 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { ElementAspect, ElementMultiAspect, ElementUniqueAspect } from "@itwin/core-backend";
+import { Id64String } from "@itwin/core-bentley";
+import { ElementAspectsHandler, ExportElementAspectsStrategy } from "./ExportElementAspectsStrategy";
+import { ensureECSqlReaderIsAsyncIterableIterator } from "./ECSqlReaderAsyncIterableIteratorAdapter";
+import { ElementAspectProps, QueryBinder, QueryRowFormat } from "@itwin/core-common";
+import { IModelExportHandler } from "./IModelExporter";
+
+/**
+ * Handler for [[DetachedExportElementAspectsStrategy]]
+ * @internal
+ */
+export interface DetachedElementAspectsHandler extends ElementAspectsHandler {
+  onExportElementUniqueAspect(uniqueAspect: ElementUniqueAspect, isUpdate?: boolean | undefined): void;
+  onExportElementMultiAspects(multiAspects: ElementMultiAspect[]): void;
+}
+
+/**
+ * Detached ElementAspect export strategy for [[IModelExporter]].
+ * This strategy exports all ElementAspects separately from the Elements that own them.
+ * @internal
+ */
+export class DetachedExportElementAspectsStrategy extends ExportElementAspectsStrategy<DetachedElementAspectsHandler> {
+  public override registerHandler(handler: () => IModelExportHandler, trackProgress: () => Promise<void>) {
+    super.registerHandler(handler, trackProgress);
+
+    this.handler.onExportElementUniqueAspect = (uniqueAspect, isUpdate) => handler().onExportElementUniqueAspect(uniqueAspect, isUpdate);
```

Comment:
handler callback here wasn't a constructor, but a getter which can return the same instance on every callback. Either way, my intention was to be able to support different element aspect strategy processing handlers so that it would be possible to change the interface and not adjust the old strategies for it, but it creates the problem on how to handle custom handler creation where either `IModelExporter` has to know each strategy and how to register its handler or must call some `registerHandler` method for strategy and provide a lazy way to retrieve `IModelExportHandler` and each handler class would register its portion of handler interface. Neither solution seems clean enough and we don't actually need it now. So I I removed custom aspect handler interfaces and if it will be needed we will be able to add it later when it's actually needed